packages/bpk-component-text/src/BpkText.module.scss

```diff
@@ -57,6 +57,74 @@
     @include bpk-text--xxxxxl;
   }
 
+  &--caption {
+    @include bpk-caption;
+  }
+
+  &--footnote {
+    @include bpk-footnote;
+  }
+
+  &--label1 {
```

Comment:
> We probably want to follow the BEM format or making this clearer by following the naming as the mixin names are for the classes for all the newly introduced styles from this one below
> 
> ```css-scss
> &--label-1
> ```

Oh right, yeah I didn't add the '-' because the keys in TEXT_STYLES need to be camel case so I kept it the same
Will change the class names though